test/Google_Api_Test/src/Main.enso

```diff
@@ -0,0 +1,18 @@
+from Standard.Base import all
+import Standard.Google_Api
+import Standard.Test
+
+main = Test.Suite.run_main <|
```

Comment:
We should probably make this `pending` if it isn't on CI.

Reply:
It's in a separate project, so just don't run it if you don't want to :P